modules/integration_azure-cdn/conf/readme.yaml

```diff
@@ -0,0 +1,3 @@
+documentations:
+
```

Comment:
Please add a link to MS doc https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/metrics-supported#microsoftcdnprofiles

Reply:
I don't see any change here :thinking: